core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -27,12 +27,13 @@ import java.util.concurrent.atomic.{AtomicReference, AtomicBoolean, AtomicIntege
 import java.util.UUID.randomUUID
 
 import scala.collection.{Map, Set}
-import scala.collection.JavaConversions._
 import scala.collection.generic.Growable
+import scala.collection.JavaConversions._
```

Comment:
well, it's very nitpicky, but `generic` is a package, `JavaConversions` is a class; imports are sorted by package only.